.rubocop_todo.yml

```diff
@@ -1551,13 +1546,7 @@ Rails/NegateInclude:
     - 'app/workers/web/push_notification_worker.rb'
     - 'lib/paperclip/color_extractor.rb'
 
-# This cop supports unsafe autocorrection (--autocorrect-all).
-# Configuration parameters: Include.
-# Include: app/**/*.rb, config/**/*.rb, db/**/*.rb, lib/**/*.rb
-Rails/Output:
-  Exclude:
-    - 'lib/mastodon/ip_blocks_cli.rb'
-
+# Offense count: 9
```

Comment:
If you run `bundle exec rubocop --regen-todo` it should generate it with the current settings that won't add these anymore
```suggestion
```